axelor-human-resource/src/main/java/com/axelor/apps/hr/service/allocation/AllocationLineComputeServiceImpl.java

```diff
@@ -211,4 +213,60 @@ protected BigDecimal computeProrata(
 
     return prorata;
   }
+
+  @Override
+  public BigDecimal getAllocatedTime(Project project, Sprint sprint) {
+    List<AllocationLine> allocationLineList =
+        allocationLineRepo.all().fetch().stream()
+            .filter(allocationLine -> allocationLine.getProject().equals(project))
+            .collect(Collectors.toList());
+    BigDecimal allocatedTime = BigDecimal.ZERO;
+    for (AllocationLine allocationLine : allocationLineList) {
+      LocalDate allocationLineFromDate = allocationLine.getPeriod().getFromDate();
+      LocalDate allocationLineToDate = allocationLine.getPeriod().getToDate();
+      LocalDate sprintFromDate = sprint.getFromDate();
+      LocalDate sprintToDate = sprint.getToDate();
+
+      int sprintPeriod = 0;
+      if (sprintFromDate.isAfter(allocationLineFromDate)
+          && sprintToDate.isBefore(allocationLineToDate)) {
+        sprintPeriod = getWeekDaysNumber(sprintFromDate, sprintToDate);
+      }
+
+      if (!sprintFromDate.isAfter(allocationLineFromDate)
+          && sprintToDate.isBefore(allocationLineToDate)) {
+        sprintPeriod = getWeekDaysNumber(allocationLineFromDate, sprintToDate);
+      }
+
+      if (sprintFromDate.isAfter(allocationLineFromDate)
+          && !sprintToDate.isBefore(allocationLineToDate)) {
+        sprintPeriod = getWeekDaysNumber(sprintFromDate, allocationLineToDate);
+      }
+
+      int allocationPeriod = getWeekDaysNumber(allocationLineFromDate, allocationLineToDate);
+      BigDecimal allocation = allocationLine.getAllocated();
+      allocatedTime = allocatedTime.add(getProrata(allocationPeriod, sprintPeriod, allocation));
+    }
+    return allocatedTime;
+  }
+
+  int getWeekDaysNumber(LocalDate startDate, LocalDate toDate) {
```

Comment:
Seems existing, AllocationLineComputeServiceImpl.getWorkingDays(), check the number of working days in a period using the employee's planning